src/redux/migrations.ts

```diff
@@ -793,15 +793,28 @@ export const migrations = {
     },
   }),
   72: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      showGuidedOnboardingCopy: REMOTE_CONFIG_VALUES_DEFAULTS.showGuidedOnboardingCopy,
```

Comment:
This is incorrect. I suspect while you were working other people made changes to redux state. You need to make a new version and not modify the old ones like this.